torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -189,7 +189,10 @@ Tensor norm_backward(const Tensor& grad, const Tensor& self, const optional<Scal
   return norm_backward(grad, self, p_, norm, {}, true);
 }
 
-Tensor norm_backward(Tensor grad, const Tensor& self, const optional<Scalar> & p_, Tensor norm, IntArrayRef dim, bool keepdim) {
+Tensor norm_backward(
+    Tensor grad, const Tensor& self, const optional<Scalar> & p_, Tensor norm, IntArrayRef dim, bool keepdim) {
+  // NB: we could've masked_fill the zero elements with an arbitrary value like 1 to
```

Comment:
I don't think this comment is precise enough. What are you maske filling? And why?

Reply:
Updated!